scoreboard/data/threads-config.yml

```diff
@@ -0,0 +1,28 @@
+scoreboard:
+  task:
+    mpi:
+      solution:
+        max: 10
+      visible: true
+    omp:
+      solution:
```

Comment:
please add max of performance hear too as same solution

Reply:
Added